modules/tensor_mechanics/test/tests/uel/small_test_expanded.i

```diff
@@ -0,0 +1,125 @@
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 20
+    ny = 10
+    # nx = 10
+    # ny = 3
```

Comment:
You've worn me down. Let's remove unused lines from these tests

Reply:
Ha. While I appreciate the use of the Github suggestions feature (and you essentially making the edits for me) it would be totally fine to simply state "please remove all commented out code and parameters".